sdk/keyvault/Azure.ResourceManager.KeyVault/src/autorest.md

```diff
@@ -44,4 +44,8 @@ directive:
         "modelAsString": false,
         "name": "NameAvailabilityReason"
       }
+  - from: swagger-document
```

Comment:
Why not use `rename-model` (like `BaseAttributes` above)?

Reply:
Throw error when trying to rename 'Resource'